main.nf

```diff
@@ -14,201 +14,183 @@ Sabrina Krakau <sabrinakrakau@gmail.com>
+////////////////////////////////////////////////////
+
+// Don't overwrite global params.modules, create a copy instead and use that within the main script.
+def modules = params.modules.clone()
+
+def multiqc_options   = modules['multiqc']
+multiqc_options.args += params.multiqc_title ? " --title \"$params.multiqc_title\"" : ''
+
+// Local: Modules
+include { GET_SOFTWARE_VERSIONS                               } from './modules/local/get_software_versions'       addParams( options: [publish_files : ['csv':'']]          )
+include { BOWTIE2_REMOVAL_BUILD as BOWTIE2_HOST_REMOVAL_BUILD } from './modules/local/bowtie2_removal_build'       addParams( options: [:]                                   )
+include { BOWTIE2_REMOVAL_ALIGN as BOWTIE2_HOST_REMOVAL_ALIGN } from './modules/local/bowtie2_removal_align'       addParams( options: modules['bowtie2_host_removal_align'] )
+include { BOWTIE2_REMOVAL_BUILD as BOWTIE2_PHIX_REMOVAL_BUILD } from './modules/local/bowtie2_removal_build'       addParams( options: [:]                                   )
+include { BOWTIE2_REMOVAL_ALIGN as BOWTIE2_PHIX_REMOVAL_ALIGN } from './modules/local/bowtie2_removal_align'       addParams( options: modules['bowtie2_phix_removal_align'] )
+include { PORECHOP                                            } from './modules/local/porechop'                    addParams( options: [:]                                   )
+include { NANOLYSE                                            } from './modules/local/nanolyse'                    addParams( options: modules['nanolyse']                   )
+include { FILTLONG                                            } from './modules/local/filtlong'                    addParams( options: [:]                                   )
+include { NANOPLOT as NANOPLOT_RAW                            } from './modules/local/nanoplot'                    addParams( options: modules['nanoplot_raw']               )
+include { NANOPLOT as NANOPLOT_FILTERED                       } from './modules/local/nanoplot'                    addParams( options: modules['nanoplot_filtered']          )
+include { CENTRIFUGE_DB_PREPARATION                           } from './modules/local/centrifuge_db_preparation'   addParams( options: [:]                                   )
+include { CENTRIFUGE                                          } from './modules/local/centrifuge'                  addParams( options: modules['centrifuge']                 )
+include { KRAKEN2_DB_PREPARATION                              } from './modules/local/kraken2_db_preparation'      addParams( options: [:]                                   )
+include { KRAKEN2                                             } from './modules/local/kraken2'                     addParams( options: modules['kraken2']                    )
+include { KRONA_DB                                            } from './modules/local/krona_db'                    addParams( options: [:]                                   )
```

Comment:
Here and others with empty options: why do you even add  `addParams( options: [:] )`? It seems to me it has no impact.

Reply:
I don't know, because others did it like this ;)

You as well https://github.com/d4straub/ampliseq/blob/9b70932af3045e0c3af9ce18456d23eabc1be593/workflows/ampliseq.nf#L159 
but seems one could omit it.

Reply:
Right, I could remove that I think. However here I did remove the addParams: https://github.com/d4straub/ampliseq/blob/9b70932af3045e0c3af9ce18456d23eabc1be593/workflows/ampliseq.nf#L149-L150